Model/Api/UniversalApi.php

```diff
@@ -125,10 +125,12 @@ public function execute(
             switch($event){
                 case "order.create":
                     //currently sends its own response updated for v2, no return
-                    $this->createOrder->execute(
-                        $event,
-                        isset($data['order']) ? $data['order'] : null,
-                        isset($data['currency']) ? $data['currency'] : null
+                    // $this->result->setData(
```

Comment:
remove redundant commented out code

Reply:
Oops, this got left commented out when I was experimenting with things. These lines should not be commented as `createOrder` now returns a value. Will fix.